CMakeLists.txt

```diff
@@ -45,6 +45,9 @@ set(FASP4NS_INCLUDE_DIR ${FASP4NS_DIR}/include)
 include_directories(${DOLFIN_INCLUDE_DIRS} ${FASP_INCLUDE_DIR} ${FASP4NS_INCLUDE_DIR} ./include)
 include_directories(SYSTEM ${DOLFIN_3RD_PARTY_INCLUDE_DIRS})
 
+# WEIRD OS X TARGET
+#set(OSX_TARGET "/opt/local/lib/gcc49/gcc/x86_64-apple-darwin14/4.9.3/libgcc.a" "/opt/local/lib/gcc49/libquadmath.a" "/opt/local/lib/gcc49/libgfortran.a")
```

Comment:
Added this line, let us use that now. You can just uncomment it if you are on OS X.